service/utils.go

```diff
@@ -0,0 +1,235 @@
+package service
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+
+	"github.com/coreos/go-semver/semver"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+
+	"github.com/giantswarm/apiextensions/pkg/clientset/versioned"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/giantswarm/operatorkit/client/k8sclient"
+	"github.com/giantswarm/operatorkit/client/k8srestconfig"
+)
+
+// Create temporary directory where all file related magic happens.
+func CreateTMPDir() (string, error) {
```

Comment:
Why bother wrapping this? Just for the log message?

Reply:
Just to easily see why there is dir creation and same for the deletion